bitrise.yml

```diff
@@ -360,6 +360,18 @@ workflows:
                 -project "$BITRISE_SOURCE_DIR/dependency-manager-tests/spm/SPMProject.xcodeproj" \
                 -destination "platform=macOS,variant=Mac Catalyst" \
                 | xcpretty
+    - script:
+        title: Check macOS compatibility (build SPMProject for macOS)
```

Comment:
❓ 
Only SPM?

Reply:
It follows what we did for Mac Catalyst. It's minimal effort for the smoke test, as the support is not official